packages/wrangler/src/dev/miniflare.ts

```diff
@@ -411,7 +411,9 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 	// Setup service bindings to external services
 	const serviceBindings: NonNullable<WorkerOptions["serviceBindings"]> = {
 		...config.serviceBindings,
-		...(config.experimentalAssets ? { ASSET_SERVER: "asset-server" } : {}),
+		...(config.experimentalAssets
+			? { ASSET_SERVER: "asset-server-worker" }
```

Comment:
```suggestion
			? { ASSET_SERVER: "asset-worker" }
```

Reply:
Can we stabilize on the same name everywhere `asset-worker` (and also change the directory name where it lives).

Reply:
yes, I agree! I will do that in a separate PR and link here

Reply:
tracking in WC-2595